jnigen/test/summary_generation_test.dart

```diff
@@ -21,13 +21,21 @@ import 'package:test/test.dart';
 
 import 'test_util/test_util.dart';
 
+const nestedClasses = [
```

Comment:
There are some more nested classes in generics/GrandParent.java.

Reply:
Done. (This is just verifying nested classes are there. So no need to be exhaustive though).